### PR TITLE
Desktop: release notes: mention kernel update and performance fix

### PR DESCRIPTION
**Pull request by thaJeztah**
relates to https://github.com/docker/for-mac/issues/5668

### CODE REVIEW

**Comment by thaJeztah** [comment]
@StefanScherer @fredericdalleau PTAL

Should we add the description to the "for win" release notes as well? Not sure if we received reports from Windows users, but if it's a kernel issue, I assume those would be equally  affected?

**Comment by StefanScherer** [comment]
@thaJeztah Yes, we can add the same description to the docker-for-win/release-notes.md

**Comment by thaJeztah** [comment]
Thanks @StefanScherer  - updated; added the note to the Windows release notes as well 👍

**Review by StefanScherer — APPROVED** [review]
LGTM
Thanks!

**Comment by thaJeztah** [comment]
FWIW: If we are good with https://github.com/docker/docker.github.io/pull/13353, I think rebasing this _after_ that one will be easier (git should in most cases automatically pick up the renamed files)

We should check if updating the URLs without changing the URLs in Docker Desktop will be a problem though (if they all open in a browser, the browser should redirect as usual)

**Comment by thaJeztah** [comment]
Rebased, as https://github.com/docker/docker.github.io/pull/13353 was merged 👍

File: desktop/mac/release-notes.md [hunk 1]
```diff
@@ -36,9 +36,14 @@ This page contains information about the new features, improvements, known issue
 - [containerd v1.4.9](https://github.com/containerd/containerd/releases/tag/v1.4.9)
 - [runc v1.0.1](https://github.com/opencontainers/runc/releases/tag/v1.0.1)
 - [Kubernetes 1.21.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.3)
+- [Linux kernel 5.10.47](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.47-0b705d955f5e283f62583c4e227d64a7924c138f/images/sha256-a4c79bc185ec9eba48dcc802a8881b9d97e532b3f803d23e5b8d4951588f4d51?context=repo)
 
 ### Bug fixes and minor changes
 
+- Update kernel configuration to fix a performance regression in [Docker Desktop 3.0.0](#docker-desktop-300)
+  that caused publishing container ports to take 10 times longer than on older
+  versions. More details can be found on [linuxkit/linuxkit#3701](https://github.com/linuxkit/linuxkit/pull/3701)
```
Comment (usha-mandya) [inline desktop/mac/release-notes.md:45]:
```suggestion
  versions. For more information, see [linuxkit/linuxkit#3701](https://github.com/linuxkit/linuxkit/pull/3701).
```

File: desktop/windows/release-notes.md [hunk 2]
```diff
@@ -35,9 +35,14 @@ This page contains information about the new features, improvements, known issue
 - [containerd v1.4.9](https://github.com/containerd/containerd/releases/tag/v1.4.9)
 - [runc v1.0.1](https://github.com/opencontainers/runc/releases/tag/v1.0.1)
 - [Kubernetes 1.21.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.3)
+- [Linux kernel 5.10.47](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.47-0b705d955f5e283f62583c4e227d64a7924c138f/images/sha256-a4c79bc185ec9eba48dcc802a8881b9d97e532b3f803d23e5b8d4951588f4d51?context=repo)
 
 ### Bug fixes and minor changes
 
+- Update kernel configuration to fix a performance regression in [Docker Desktop 3.0.0](#docker-desktop-300)
+  that caused publishing container ports to take 10 times longer than on older
+  versions. More details can be found on [linuxkit/linuxkit#3701](https://github.com/linuxkit/linuxkit/pull/3701)
```
Comment (usha-mandya) [inline desktop/windows/release-notes.md:44]:
```suggestion
  versions. For more information, see [linuxkit/linuxkit#3701](https://github.com/linuxkit/linuxkit/pull/3701)
```

**Comment by usha-mandya** [comment]
@thaJeztah Left a minor suggestion. Looks good otherwise.

**Comment by thaJeztah** [comment]
Updated! Thanks for reviewing 🤗

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**